Assets/Scripts/Background/ScaleBackgroundOld.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//
+// Old script for scaling the background (no longer used)

```

Comment:
I think the unused scripts can be removed.

Reply:
Gotcha; I deleted those now.